EngineLayer/spectralLibrarySearch/spectralLibraryReader.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Proteomics.Fragmentation;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace EngineLayer.SpectralLibrarySearch
+{
+    public static class SpectralLibraryReader
+    {
+
+        public static Dictionary<string, SpectralBook> ReadSpectralLibrary(string filePath)
+        {
+            Dictionary<string, SpectralBook> spectralLibraryDictionary = new Dictionary<string, SpectralBook>();
+            string[] lines;
+
+            lines = File.ReadAllLines(filePath);
```

Comment:
Please comment here: The efficiency might be improved with Stream in the future. Two for loops are used but only one is needed if combine them togehter.